src/shared/components/Searchbar/Searchbar.style.tsx

```diff
@@ -32,17 +32,11 @@ export const CancelButton = styled(Button)`
   position: absolute;
```

Comment:
Maybe we could add some margin-right here? Seems like the button is too closely hugging the right border